frontend/vite.config.js

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from "vite";
+import solidPlugin from "vite-plugin-solid";
+
+export default defineConfig({
+  plugins: [solidPlugin()],
+  server: {
+    port: 3000,
```

Comment:
Please, set port using the environmental variable

Reply:
done